src/PID/MotorControllerEnhancedPIDController.java

```diff
@@ -0,0 +1,87 @@
+package PID;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.IMotorControllerEnhanced;
+
+public class MotorControllerEnhancedPIDController implements PIDController {
+  private IMotorControllerEnhanced motorControllerEnhanced;
+  private PIDFTerms pidfTerms;
+  private int PIDSlot;
```

Comment:
think it should be pidSlot according to conventions? put pid as an abbreviation and intelliJ should tell you.